### DISCUSSION TITLE
More than 10 categories?

### DISCUSSION THREAD

**Discussion by albertkun** [community/community#3793] [Discussions]
Hello! I am using GitHub discussion for organizing weekly class assignments and discussions and I really like it!!

![image](https://user-images.githubusercontent.com/8574425/114958780-94ee9700-9e18-11eb-9abc-d62b9439bfd0.png)

However I am afraid I will hit the 10 category limit. Would it be possible to add more than 10 categories? Thank you!

**Comment by kimploo** [comment]
My company is also waiting for more than 10 categories. This feature has been huge heap for our service for students of our web dev bootcamp. If we can manage more than 10 categories, then our students can debate in detail, find appropriate questions and answers for their detailed need. Waiting for your support! Thank you for this awesome project and project team!

**Comment by laymonage** [comment]
With the new labels feature, how about adding separate labels for each week? They'll be in the same category (e.g. Weekly), but you can assign a label for the week, e.g. Week 3.
